opi/__init__.py

```diff
@@ -248,7 +261,12 @@ def install_binary(binary):
 			gpgcheck = True,
 			auto_refresh = True
 		)
-		install_packages([name_with_arch], from_repo=project, flags=['--allow-vendor-change', '--allow-arch-change', '--allow-downgrade', '--allow-name-change'])
+		flags = []
+		if get_backend() == BackendConstants.zypp:
+			flags = ['--allow-vendor-change', '--allow-arch-change', '--allow-downgrade', '--allow-name-change']
+		elif get_backend() == BackendConstants.dnf:
+			flags = ['--setopt=allow_vendor_change=True'] # TODO: architecture changes, name changes, downgrades?
```

Comment:
Please make sure to find alternatives to these flags from zypper.
Also it might make sense to not use backend specific flag as args to the `install_packages` function anymore but normal kwargs like `allow_vendor_change=False`. This way all the backend specifics could be handled in `install_packages` instead of in `install_packages`.

Reply:
Moving the flags to `install_packages()` is a good idea, will do that. `--allow-downgrade` and `--allow-name-change` are, according to @Conan-Kudo, automatically active in dnf. For arch changes, we need to specify the target architecture, so we need to somehow figure that out.